crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -653,6 +648,25 @@ impl RealRibosome {
                 get_validation_receipts,
             );
 
+        #[cfg(feature = "unstable-functions")]
+        host_fn_builder
+            .with_host_function(
+                &mut ns,
+                "__hc__accept_countersigning_preflight_request_1",
+                accept_countersigning_preflight_request,
+            )
+            .with_host_function(
+                &mut ns,
+                "__hc__get_agent_key_lineage_1",
+                get_agent_key_lineage,
+            )
+            .with_host_function(&mut ns, "__hc__is_same_agent_1", is_same_agent)
+            .with_host_function(&mut ns, "__hc__block_agent_1", block_agent)
+            .with_host_function(&mut ns, "__hc__schedule_1", schedule)
+            .with_host_function(&mut ns, "__hc__unblock_agent_1", unblock_agent)
+            // TODO deprecated, remove me
```

Comment:
Why not remove it now?

Reply:
See the description, it breaks more than 150 DPKI tests

Reply:
Why do I ask this twice?

Reply:
I did put a TODO in several places :D